csharp.benchmark/Check.cs

```diff
@@ -21,9 +21,10 @@ public static void ArraysAreEqual<T>(T[] expected, T[] result)
             {
                 if (!comparer.Equals(expected[i], result[i]))
                 {
-                    throw new InvalidDataException($"expected value {expected[i]} != result value {result[i]} at index {i}");
+                    throw new InvalidDataException(
+                        $"expected value {expected[i]} != result value {result[i]} at index {i}");
```

Comment:
Here is an example of a line that got split into two lines, probably because it was 'too long' according to the current settings (just a guess)

Reply:
finally got it! please, when you can enable the latest CI
FYI I found out that there is no way to generate `ParquetSharp.DotSettings` via some CLI. The only way to do it is to import the project in ReSharper IDE and check which options you would like to have (Code Style -> C#). In docc they are referencing this concept as [Layers](https://www.jetbrains.com/help/rider/Sharing_Configuration_Options.html). This file was generated as part of [Solution-Team shared](https://www.jetbrains.com/help/rider/Sharing_Configuration_Options.html#solution-team-shared-layer) layer.
Let me know if you found any other instances, of irregularity since with example I can easily find and match which rule needs to be changed. 
Thank you guys! @GPSnoopy @jgiannuzzi 🍻